Assets/Mirage/Runtime/NetworkSceneManager.cs

```diff
@@ -182,210 +173,468 @@ internal void OnClientChangeScene(string scenePath, SceneOperation sceneOperatio
         /// </summary>
         /// <param name="scenePath">Path of the scene that was just loaded</param>
         /// <param name="sceneOperation">Scene operation that was just  happen</param>
-        internal void OnClientSceneChanged(string scenePath, SceneOperation sceneOperation)
+        internal void OnClientSceneLoadFinished(string scenePath, SceneOperation sceneOperation)
         {
-            if (pendingAdditiveSceneList.Count > 0 && Client && !Client.IsLocalClient)
+            if (ClientPendingAdditiveSceneLoadingList.Count > 0 && Client && !Client.IsLocalClient)
             {
-                ApplyOperationAsync(pendingAdditiveSceneList[0], SceneOperation.LoadAdditive).Forget();
-                pendingAdditiveSceneList.RemoveAt(0);
+                LoadSceneAsync(ClientPendingAdditiveSceneLoadingList[0], new[] {Client.Player}, SceneOperation.LoadAdditive).Forget();
```

Comment:
why do the scene need to be loaded 1 at a time?

Reply:
good catch changed it to just load the range,